ui/src/app/app-routing.module.ts

```diff
@@ -155,28 +156,29 @@ const routes: Routes = [
                     children: [
                       {
                         path: '',
-                        component: NotebookListComponent,
+                        component: AppRouting,
                         data: {
-                          title: 'View Notebooks',
                           breadcrumb: BreadcrumbType.Workspace,
                           helpContentKey: 'notebooks'
```

Comment:
The ticket includes dealing with these two parameters. Do we no longer intend to move these over piecemeal?

Reply:
Yes, I added reviewers and took this out of draft state prematurely - I noticed this as I was typing up some notes in the description. I will take a look at this.